### DISCUSSION TITLE
How do you extract Stereo (Left and Right) signal from the FFT

### DISCUSSION THREAD

**Discussion by Knottis** [pschatzmann/arduino-audio-tools#878] [Q&A] [answered]
### **Hello!**

I receives Audio signal from the BluetoothA2DPSink and then I wants to extract Left and Right signal from the FFT.

_Haven't found any examples of this so I hope this is possible an there is a simple solution?._

_**Thank you so much!**_

**Comment by pschatzmann** [comment, marked as answer]
FFT is a very expensive operation (both using a lot of memory and doing a lot of floating point operations), I will never do it on a stereo signal and if you feed it with stereo, it is converted to a mono signal first.
If you really want do do this you will need to split up the signal yourself and you will need to set up 2 FFTs.
